provider/pkg/provider/helm_release.go

```diff
@@ -1096,6 +1073,42 @@ func computeResourceNames(r *Release) (map[string][]string, error) {
 	return resourceNames, nil
 }
 
+func (r *helmReleaseProvider) chartOptsFromRelease(rel *Release) HelmChartOpts {
+	helmHome := os.Getenv("HELM_HOME")
+
+	helmChartOpts := HelmChartOpts{
+		APIVersions:              nil,
+		SkipCRDRendering:         rel.SkipCrds,
+		Namespace:                rel.Namespace,
+		ReleaseName:              rel.Name,
+		Values:                   rel.Values,
+		Version:                  rel.Version,
+		HelmChartDebug:           r.settings.Debug,
+		IncludeTestHookResources: true,
```

Comment:
Outside the scope of this PR, but we should probably expose this as an option for users